migrations/20171114160600_build_step.js

```diff
@@ -0,0 +1,19 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('build_step', function(table) {
+    table.uuid('id').index().unique().notNullable();
+    table.uuid('buildId').index().notNullable();
+    table.integer('order').notNullable();
+    table.text('description');
+    table.string('name');
+    table.string('plugin');
+    // @todo: do we want this to be an enum, or leave it open to more statuses?
```

Comment:
My inclination is to just leave it open so that new plugins created in the main `probo` project don't also need a sibling PR in this project too.

Reply:
+1